docs/spec.md

```diff
@@ -5091,12 +5091,10 @@ Performs element-wise square root operation on `operand` tensor and produces a
 // %operand: [[0.0, 1.0], [4.0, 9.0]]
```

Comment:
Please update the PR description to follow the testing guidelines, e.g. similarly to #1080.